akka-stream-tests/src/test/scala/akka/stream/scaladsl/RestartSpec.scala

```diff
@@ -698,58 +699,54 @@ class RestartSpec
     }
 
     "restart on cancellation" taggedAs TimingTest in {
-      val (created, source, flowInProbe, flowOutProbe, sink) = setupFlow(shortMinBackoff, shortMaxBackoff)
+      val (created, source, inProbe, flowOutProbe, sink) = setupFlow(shortMinBackoff, shortMaxBackoff)
 
       source.sendNext("a")
-      flowInProbe.requestNext("a")
+      inProbe.expectMsg("a")
```

Comment:
I think you should have kept it `flowInProbe` because at least that gives a hint about what it probes.

Reply:
ok, I will change the name.

Reply:
done